ministryofjustice/laa-review-criminal-legal-aid
# Crimap 586 view latest full month

## Description of change
Show the sample caseworker report for the latest full month.

## Link to relevant ticket
**[CRIMAP-586](https://dsdmoj.atlassian.net/browse/CRIMAP-586)**

## Notes for reviewer

Unit testing will be added when the weekly and daily reports have been added.
The dashboard view is completely undesigned.

## Screenshots of changes (if applicable)

### Before changes:
<img width="771" alt="Screenshot 2023-09-13 at 16 45 46" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/34935/0b1504c1-f7ce-4df2-bdc5-219879e38481">


### After changes:
<img width="797" alt="Screenshot 2023-09-13 at 16 45 30" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/34935/24cd0b54-feaa-46f2-8cfd-0587774e7d19">

<img width="623" alt="Screenshot 2023-09-14 at 12 06 16" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/34935/9d8b5150-9d81-464e-9f58-ab8cfd9983fc">

## How to manually test the feature

Create a DataAnalyst user. Log in. A link to the latest full month should appear on landing page.


[CRIMAP-586]: https://dsdmoj.atlassian.net/browse/CRIMAP-586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
